std_msgs/Header header
sensor_msgs/RegionOfInterest[] bboxes
recycling_stretch/RotatedBBox[] rbboxes
float64[] scores
int64[] pred_category